ethcore/vm/src/action_params.rs

```diff
@@ -96,6 +96,8 @@ pub struct ActionParams {
 	pub params_type: ParamsType,
 	/// The Oasis contract extracted from the code (if header is present).
 	pub oasis_contract: Option<OasisContract>,
+	/// The AAD (Additional Authenticated Data) of confidential execution.
```

Comment:
Would be helpful to note that this is the AAD from the confidential transaction/request.